resources/known-amendments.md

```diff
@@ -982,6 +1042,23 @@ Simplifies and unifies the code for decimal floating point math. In some cases,
 Without this amendment, the code continues to use separate calculations for `STAmount` and `IOUAmount` objects, and [Automated Market Maker (XLS-30d)](https://github.com/XRPLF/XRPL-Standards/discussions/78) uses a third class for calculations.
 
 
+### fixXChainRewardRounding
+[fixXChainRewardRounding]: #fixxchainrewardrounding
+
+| Amendment    | fixXChainRewardRounding |
+|:-------------|:------------------------|
+| Amendment ID | 2BF037D90E1B676B17592A8AF55E88DB465398B4B597AE46EECEE1399AB05699 |
+| Status       | Open for Voting |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+This amendment ensures that reward shares for cross-chain transactions are always rounded down.
+
+Without this amendment, if the `fixUniversalNumber` amendment is enabled, reward amounts could be rounded up in ways that cause problems.
```

Comment:
Here's a first stab at text to describe what the `fixXChainRewardRounding` amendment does...

This amendment ensures that reward shares for cross-chain transactions are always rounded down.  This preserves the original intended behavior.

The `fixUniversalNumber` amendment introduced a new numeric type into the code base: Number.  The default rounding behavior for Number is "to nearest".  When the `fixUniversalNumber` amendment is enabled, Number is used for some intermediate calculations of IOU amounts.  That means values that used to round down now round to nearest.  This newly introduced rounding behavior alters the intended rounding behavior for reward shares in cross-chain transactions.

The `fixXChainRewardRounding` amendment corrects the oversight.  So now when reward shares are calculated they will, once again, always round down.